### PR TITLE
init commit

### PR DESCRIPTION
**Pull request by MEtoCS**

### CODE REVIEW

File: com.mashibing.study/src/main/java/controller/MyController.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package controller;
+
+public class MyController {
+    //this is a controller
```
Comment (longchenlongchenlongchen) [inline com.mashibing.study/src/main/java/controller/MyController.java:4]: Looks good

File: com.mashibing.study/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst [hunk 2]
```diff
@@ -0,0 +1 @@
+controller\MyController.class
```
Comment (longchenlongchenlongchen) [inline com.mashibing.study/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst:1]: Good name convention

Comment (MEtoCS) [reply com.mashibing.study/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst:1]: Got it

File: com.mashibing.study/src/main/java/controller/MyController.java (hunk 1 above)
Comment (MEtoCS) [reply com.mashibing.study/src/main/java/controller/MyController.java:4]: alright update

File: com.mashibing.study/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst (hunk 2 above)
Comment (longchenlongchenlongchen) [reply com.mashibing.study/target/maven-status/maven-compiler-plugin/compile/default-compile/createdFiles.lst:1]: Approve it

**Review by longchenlongchenlongchen — APPROVED** [review]
Can be merged

**Pull request merged**